### PR TITLE
LPS-114787 Support mime type size limits at site level

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by liuxingye** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/configuration/DLSizeLimitConfiguration.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
  */
 @ExtendedObjectClassDefinition(
 	category = "documents-and-media",
-	scope = ExtendedObjectClassDefinition.Scope.COMPANY
+	scope = ExtendedObjectClassDefinition.Scope.GROUP
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/configuration/DLSizeLimitConfiguration.java:26]: maybe rename the class to DLSizeLimitGroupConfiguration it's a good idea?

**Review by AliciaGarciaGarcia — COMMENTED** [review]
The only thing I can add is if it's maybe good idea to change the builder of DLStoreRequest adding the groupId instead of use _GroupThreadLocal.getGroupId()_
It's a Breaking Change but now we are using it ...

**Comment by Hanlf** [comment]
[com.liferay.journal.asset.test.JournalArticleAssetSearchTest.testOrderByCreateDateAsc](https://testray.liferay.com/home/-/testray/cases/1518349807) failure is not related to this PR.

**Comment by Hanlf** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
MySQL docker issue, not related to this PR.

**Comment by Hanlf** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**